backend/api/Database/Models/Report.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+# nullable disable
+namespace Api.Models
+{
+    public class Report
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Required]
+        public string Id { get; set; }
+        [Required]
+        public virtual Robot Robot { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string IsarMissionId { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string EchoMissionId { get; set; }
+        [MaxLength(128)]
+        [Required]
+        public string Log { get; set; }
+        [Required]
+        public ReportStatus ReportStatus { get; set; }
+        [Required]
+        public DateTimeOffset StartTime { get; set; }
+        public DateTimeOffset EndTime { get; set; }
+        [Required]
+        public virtual ICollection<ReportEntry> Entries { get; private set; }
+    }
+
+    public enum ReportStatus
+    {
+        InProgress, Completed, Failed
+    }
```

Comment:
Have a look at this file in my PR: https://github.com/equinor/flotilla/pull/194/files#diff-99e3bfb79646488558a4630ec2f67093c1ed92ca696cb415b7f781a45ca43cccR1-R21
Personally I'm a fan of the enum with string values for these types of enums because the values themselves will be written to the database and provided to the user through json. If we return this enum the entries will be integers rather than `"in_progress"` for example.

Reply:
Can we do something like this instead?
https://avarnon.medium.com/how-to-show-enums-as-strings-in-swashbuckle-aspnetcore-628d0cc271e6
(may be a smooth way out there), I really dont wanna make 100 lines of code for each enum

Reply:
Solved it by adding a json serializer to the controller:
`builder.Services.AddControllers().AddJsonOptions(options => options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));`